src/components/Table/StatefulTable.test.jsx

```diff
@@ -24,6 +24,28 @@ describe('stateful table with real reducer', () => {
     expect(screen.queryByDisplayValue('option-B')).not.toBeInTheDocument();
   });
 
+  it('should update aria label', () => {
+    const statefulTable = (
+      <StatefulTable
+        {...initialState}
+        options={{ hasFilter: false, hasRowExpansion: false, hasResize: true }}
+        actions={mockActions}
+      />
+    );
+    render(statefulTable);
```

Comment:
The component can be placed directly inside `render`

```suggestion
    render(
      <StatefulTable
        {...initialState}
        options={{ hasFilter: false, hasRowExpansion: false, hasResize: true }}
        actions={mockActions}
      />
    );
```